.circleci/config.yml

```
@@ -11,7 +11,7 @@ jobs:
           command: cargo fmt --all -- --check
       - run:
           name: "run lint"
-          command: cargo clippy --all -- -Dclippy::all
+          command: cargo clippy --all --features=wasm_next -- -Dclippy::all
```

```suggestion
          command: cargo clippy --all --all-features -- -Dclippy::all
```

So there is no risk of forgetting to test new features if they are added

---

I don't want to do that, necessarily. I'd rather be explicit with what needs to be tested.

---

Since features are (should be) additive, `--all-features` should just mean testing all the code :shrug: . If we want to make sure that code still compiles without some features I like to run `cargo check --no-default-features` as a second step then.

---

I am mostly using feature flags as a way to merge things into master without breaking the world, and less for what the rust community uses them for. When `wasm_next`, as a feature, can be deleted, then I know the feature is complete. I _suspect_ as we get closer to getting this into the UI, we'll put the old API behind `cfg(not(feature = "wasm_next"))` which is _also_ not a way to use rust feature flags, but our scope is currently so limited, and this isn't a crate that gets put up on crates.io, so we have a little...interpretive wiggle room here.